pkg/arvo/sys/hoon.hoon

```diff
@@ -5455,9 +5455,10 @@
   ~%  %co  ..co  ~
   =<  |_  lot=coin
       ++  rear  |=(rom=tape rend(rep rom))
-      ++  rent  `@ta`(rap 3 rend)
+      ++  rent  ~+  `@ta`(rap 3 rend)
       ++  rend
         ^-  tape
+        ~+
```

Comment:
I'm a little hesitant to have both of these arms memoized; it seem like it could be a lot of churn in the cache. I'm also curious to see the impact of #4692 on these numbers. (Also, we have a space leak on arvo reset, so I really want to avoid frequent hoon/arvo changes. But that has to be fixed regardless.)

Reply:
In particular, these are frequently called on dates, which are new per event. Not the ideal cache candidate.